src/dash/models/DashManifestModel.js

```diff
@@ -203,6 +203,14 @@ function DashManifestModel() {
                 return;
             }
 
+            if (prt.hasOwnProperty(DashConstants.INBAND)) {
+                entry[DashConstants.INBAND] = prt[DashConstants.INBAND] !== 'false';
```

Comment:
Comparing to a string is correct here, type is `type="xs:boolean"`

Reply:
Reference stream contains info as string, closing this comment

Reply:
and boolean values are not converted into boolean types in parser (same for availabilityTimeComplete for example)